
From: David Howells <dhowells@redhat.com>

The attached patch exports file_ra_state_init() again for use by CacheFS. 
This is required as that calls do_generic_mapping_read() directly on
inodes.

Signed-Off-By: David Howells <dhowells@redhat.com>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/mm/readahead.c |    1 +
 1 files changed, 1 insertion(+)

diff -puN mm/readahead.c~export-file_ra_state_init-again mm/readahead.c
--- 25/mm/readahead.c~export-file_ra_state_init-again	2005-01-10 21:15:02.093135376 -0800
+++ 25-akpm/mm/readahead.c	2005-01-10 21:15:02.097134768 -0800
@@ -37,6 +37,7 @@ file_ra_state_init(struct file_ra_state 
 	ra->ra_pages = mapping->backing_dev_info->ra_pages;
 	ra->prev_page = -1;
 }
+EXPORT_SYMBOL_GPL(file_ra_state_init);
 
 /*
  * Return max readahead size for this inode in number-of-pages.
_
